apps/admin_app/lib/admin_app_web/views/product_view.ex

```diff
@@ -36,6 +49,16 @@ defmodule AdminAppWeb.ProductView do
     length(product.taxon.variation_themes) > 0
   end
 
+  def is_variant_tracking_enabled(product) do
```

Comment:
@SagarKarwande are you sure about placing this method in the `product_view`? can you justify the responsibility of the file and the this method?

Reply:
I think we can move it to `Schema level` as it is related to Enum. Changing the comparison value would be quite easy later on.